mod-source-record-storage-server/src/test/java/org/folio/rest/impl/SampleRecordsTest.java

```diff
@@ -26,7 +26,7 @@ public void shouldReturnListWithSampleData() {
       .get(SOURCE_STORAGE_SOURCE_RECORDS_PATH)
       .then()
       .statusCode(HttpStatus.SC_OK)
-      .body("totalRecords", is(11))
+      .body("totalRecords", is(10))
```

Comment:
If I'm not mistaken, we have 11 sample records. Why did you change the expected number?